src/shipit_code_coverage/shipit_code_coverage/report_generators.py

```diff
@@ -39,19 +39,18 @@ def zero_coverage(artifacts, out_dir='code-coverage-reports'):
         if all_lines_uncovered or (len(sf['functions']) > 1 and all_functions_uncovered):
             zero_coverage_files.append(name)
 
-    with open(os.path.join(out_dir, 'zero_coverage_files.json'), 'w') as f:
-        json.dump(zero_coverage_files, f)
-
     mkdir(os.path.join(out_dir, 'zero_coverage_functions'))
 
     zero_coverage_function_counts = []
```

Comment:
Let's rename this to `zero_coverage_report`, to match the name of the output file.